### DISCUSSION TITLE
[blocks]: Navbar

### DISCUSSION THREAD

**Discussion by JayanthGummitha** [shadcn-ui/ui#4626] [Blocks Request]
### Description

A Navbar Blocks that contains all features like Logo,  Menu items, Search box, signup/login button, notification,  Avatar, toggle 

OR
please provide the Navbar components..
### Example

https://www.material-tailwind.com/docs/react/navbar

**Comment by nwvh** [comment]
There's already blocks with navbars/sidebar included. Isn't that enough?

**Comment by bherila** [comment]
> There's already blocks with navbars/sidebar included. Isn't that enough?

The blocks with sidebar aren't really enough. I think it would be nice to have a top navbar block with the Menu as provided, but also a theme toggle, branding logo, and user login/logout affordances.

> A Navbar Blocks that contains all features like Logo, Menu items, Search box, signup/login button, notification, Avatar, toggle


There's one here https://www.shadcnblocks.com/block/navbar1/# (minus avatar + theme switch)

> please provide the Navbar components..

The shadcn website itself is open sourced, you can find the navbar code in there. Warning: It's fairly advanced.
